src/constants/condition.js

```diff
@@ -0,0 +1,29 @@
+const LOTTO_CONDITION = {
+  lottoPrice: 1000,
+  lottoNumberMinRange: 1,
+  lottoNumberMaxRange: 45,
+  lottoDigits: 6,
+};
+
+const PRIZE_MATCH_COUNT = Object.freeze({
+  firstPrize: 6,
+  secondPrize: 5,
+  thirdPrize: 5,
+  fourthPrize: 4,
+  fifthPrize: 3,
+});
+
+const LOTTO_PRIZE_MONEY = Object.freeze({
+  firstPrize: 2000000000,
+  secondPrize: 30000000,
+  thirdPrize: 1500000,
+  fourthPrize: 50000,
+  fifthPrize: 5000,
```

Comment:
```suggestion
  firstPrize: 2_000_000_000,
  secondPrize: 30_000_000,
  thirdPrize: 1_500_000,
  fourthPrize: 50_000,
  fifthPrize: 5_000,
```

numeric separator 를 쓰거나 지수표기법을 쓰면 자리수를 표현할 수 있어서 더 가독성이 높아집니다 😄